src/util.rs

```diff
@@ -41,15 +42,22 @@ use std::{
 /// Check whether the current rust channel is valid: `nightly` is recommended.
 pub fn assert_channel() -> Result<()> {
     let meta = rustc_version::version_meta()?;
+    let min_version = Version::new(1, 63, 0);
     match meta.channel {
-        Channel::Dev | Channel::Nightly => Ok(()),
-        Channel::Stable | Channel::Beta => {
+        Channel::Stable if meta.semver >= min_version => Ok(()),
+        Channel::Stable => {
+            anyhow::bail!(
+                "Minimum Rust version is {}. You are using {}.",
+                min_version,
+                meta.semver
+            )
+        }
+        _ => {
             anyhow::bail!(
                 "cargo-contract cannot build using the {:?} channel. \
-                Switch to nightly. \
-                See https://github.com/paritytech/cargo-contract#build-requires-the-nightly-toolchain",
+                Contracts should be build using a stable toolchain.",
```

Comment:
```suggestion
                Contracts should be build using a "stable" toolchain.",
```

Matches the `"nightly"` output from the line before